pkgs/by-name/sc/scipopt-zimpl/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  bison,
+  flex,
+  gmp,
+  zlib,
+}:
+
+stdenv.mkDerivation {
+  pname = "scipopt-zimpl";
+  version = "362";
+
+  src = fetchFromGitHub {
+    owner = "scipopt";
+    repo = "zimpl";
+    rev = "v362";
+    sha256 = "juqAwzqBArsFXmz7L7RQaE78EhQdP5P51wQFlCoo7/o=";
+  };
+
```

```suggestion
  postPatch = ''
    chmod +x check/check.sh
  '';
```
Required for last suggestion